.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+# What does this change intend to accomplish?
+
+ 
+### All Submissions:
+
+* [ ] Have you followed the guidelines in our Contributing document?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### New Feature Submissions:
+
```

Comment:
NOTE:  ./scripts/test does both of these things!

Reply:
I made a note in the testing line to try `scripts/test`; it'll also help us keep that script updated going forward :)